newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -131,12 +132,19 @@ export const getGameUrl = (game: ?Game) => {
   return GDevelopGamesPlatform.getGameUrl(game.id);
 };
 
+export const getAclsFromOwnerIds = (
```

Comment:
this new function is doing the same as below, any reason to create it rather than keep the one before and specify the "feature"?

Reply:
Actually you don't even need the feature, as you're specifying after the return of the function.
So maybe this could just become `getAclsFromUserIds()` ?